publishing-service/src/main/avro/Product.avsc

```diff
@@ -17,7 +17,7 @@
       ]
     } },
     { "name": "deliveryPromise", "type": {
-        "type": "enum", "name": "DeliveryPromise", "symbols" : [ "AVAILABLE", "NOT_AVAILABLE", "SOLD_OUT", "PREORDER" ]
+        "type": "enum", "name": "DeliveryPromise", "symbols" : [ "AVAILABLE", "NOT_AVAILABLE", "SOLD_OUT", "PREORDER", "UNKNOWN" ]
```

Comment:
This new option looks harmless but it is actually not forward compatible: Consumers need to upgrade first. The CDC breaks for those teams that require an upgrade before the producer (this publishing service) can upgrade its schema.